speed-dreams/source-2.2.3/src/modules/sound/snddefault/SoundInterface.cpp

```diff
@@ -216,6 +219,20 @@ void SoundInterface::setInterventionSound(InterventionAction p_soundEvent, const
     intervention_sounds[p_soundEvent] = sound;
 }
 
+/// @brief  Retrieves the volume of the intervention sounds
+/// @return The volume
+float SoundInterface::GetInterventionVolume() const
+{
+    return silent ? 0 : InterventionVolume;
+}
+
+/// @brief        Sets and clamps the volume of the interventions
+/// @param volume The raw volume to set
+void SoundInterface::SetInterventionVolume(float volume)
+{
+    InterventionVolume = Clamp(volume, 0.0f, 1.0f);
+}
```

Comment:
```suggestion
// SIMULATED DRIVING ASSISTANCE
/// @brief  Retrieves the volume of the intervention sounds
/// @return The volume
float SoundInterface::GetInterventionVolume() const
{
    return silent ? 0 : InterventionVolume;
}

// SIMULATED DRIVING ASSISTANCE
/// @brief        Sets and clamps the volume of the interventions
/// @param volume The raw volume to set
void SoundInterface::SetInterventionVolume(float volume)
{
    InterventionVolume = Clamp(volume, 0.0f, 1.0f);
}
```
Maybe also add to the change comment what has changed